tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1537,3 +1537,64 @@ function assertIncludesMessage(messageArr, message) {
     return false
   }
 }
+
+Then('the decline action button should be visible on the shared-with-me page', function () {
+  return batchActionButtonForSharedWithMePageIsDisplayed()
+})
+
+const batchActionButtonForSharedWithMePageIsDisplayed = function () {
+  const batchActionButtonForSharedPageXpath =
+    client.page.sharedWithMePage().elements.batchDeclineSharesButton.selector
+  return client.expect.element(batchActionButtonForSharedPageXpath).to.be.present
+}
```

Comment:
I think this function can be added to `sharedWithMePage` where you can directly access the element and you can also use `waitForElementPresent` or `waitForElementVisible`